devito/ir/iet/visitors.py

```diff
@@ -233,6 +135,9 @@ def _args_decl(self, args):
         """Convert an iterable of :class:`Argument` into cgen format."""
```

Comment:
random question: is this docstring now obsolete , in the sense that it's not just ``Argument``?

Reply:
No. It continues to be accurate.